docs/dashboards/Host Dashboard.md

```diff
@@ -0,0 +1,93 @@
+# OpenTelemetry Host Dashboard
+
+> [!WARNING]
+> The dashboards shared in this repository are in an alpha state and can change significantly.
+> They are provided as-is, with no support guarantees. 
+> Newer versions of these dashboards could look significantly different from earlier versions and add or remove certain metrics.
+
+This folder contains a dashboards that can be used to monitor visualize Hosts based of metrics ingested via the OpenTelemetry collectors using the `hostmetrics` recevier and `resourcedetection` processor. The dashboard is in JSON format and can be uploaded to your Dynatrace tenant by [following the steps in the Dynatrace documentation](https://docs.dynatrace.com/docs/shortlink/dashboards-use#dashboards-upload).
+
+![A screenshot of the host dashboard providing an overview of used system resources](img/host-dashboard_1.png)
+
+## Prerequisites
+
+Dynatrace accepts metrics data with Delta temporality via OTLP/HTTP.
+Collector and Collector Contrib versions 0.107.0 and above as well as Dynatrace collector versions 0.12.0 and above support exporting metrics data in that format.
+Earlier versions ignore the `temporality_preference` flag and would, therefore, require additional processing (cumulative to delta conversion) before ingestion.
+It is possible to do this conversion in a collector, but it would make the setup more complicated, so it is initially omitted in this document.
```

Comment:
Not sure if this sentence really brings any value. We could also remove it I think.

Reply:
```suggestion
```